src/index.js

```diff
@@ -0,0 +1,273 @@
+/* eslint no-use-before-define: 0, no-extra-parens: 0 */
```

Comment:
Why would you turn this off? This is even more important with let/const than it was with var